Backend/DataAccessLayer/Database/dbContext.cs

```diff
@@ -1,28 +1,36 @@
-﻿using DataAccessLayer.Models;
-using System;
-using System.Collections.Generic;
-using System.Data.Entity;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Microsoft.SqlServer.Server;
-using System.Data.SqlClient;
-
-namespace DataAccessLayer.Database
-{
-    public class DatabaseContext : DbContext
-    {
-        public DatabaseContext()
-        {
-            this.Database.Connection.ConnectionString = "Data Source=localhost;Initial Catalog=SSO;Integrated Security=True";
-        }
-
-        public DbSet<User> Users { get; set; }
-        public DbSet<Session> Sessions { get; set; }
-        public DbSet<Application> Applications { get; set; }
-        public DbSet<ApiKey> Keys { get; set; }
-        public DbSet<PasswordReset> ResetIDs { get; set; }
-
-
-    }
-}
+﻿using DataAccessLayer.Models;
+using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.Server;
+using System.Data.SqlClient;
+
+namespace DataAccessLayer.Database
+{
+    public class DatabaseContext : DbContext
+    {
+        string username = "Admin";
+        string password = "";
+        string hostname = "mydbinstance.ce5cmkuh7zii.us-east-2.rds.amazonaws.com";
+        string port = "1433";
+        string dbname = "mydbinstance";

```

Comment:
These variables should come from environment variables (preferable) or a config file.

Reply:
You resolved this, but it doesn't look like it's resolved.

Reply:
Added it to a config file

Reply:
See my comment - still not resolved